kubejs/startup_scripts/tfg/items.js

```diff
@@ -12,6 +12,8 @@ const registerTFGItems = (event) => {
 	event.create('tfg:unfinished_basic_electronic_circuit', 'create:sequenced_assembly')
 	event.create('tfg:unfinished_inscriber_accumulation_press')
 	event.create('tfg:vitrified_pearl')
+	event.create('tfg:cryo_fluix_pearl')
+		.translationKey('tfg:cryo_fluix_pearl')
```

Comment:
just so you know, if the translation key is identical to the item, you don't need to specify it here